src/components/Layout/PageLayout.js

```diff
@@ -6,7 +6,7 @@ import { Container } from "reactstrap"
 const Layout = ({ children }) => (
     <>
         <Nav />
-        <Container className="pt-5">{children}</Container>
+        <Container className="d-flex flex-column align-items-center">{children}</Container>
```

Comment:
Revert this back to how it was.

```suggestion
                <Container className="pt-5">{children}</Container>
```